data/sql_updates/large_aggregates.sql

```diff
@@ -123,7 +123,43 @@ party_totals as (
     group by
         month,
         year
-), -- merge
+),
+communication_totals as (
+  select
+    get_cycle(rpt_yr) as cycle,
+    f.rpt_yr as year,
+    extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+    sum(coalesce(ttl_communication_cost ,0)) as comms_totals
+  from public.fec_vsum_f7_vw f
+  where rpt_yr >= 2007
+  and most_recent_filing_flag = 'Y'
+  group by get_cycle(rpt_yr), f.rpt_yr, extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD'))
+),
+electioneering_totals as (
+  select
+    get_cycle(rpt_yr) as cycle,
+    f.rpt_yr as year,
+    extract(month from to_date(cast(end_cvg_dt as text), 'YYYY-MM-DD')) as month,
+    sum(coalesce(ttl_dons_this_stmt ,0)) as total_donations_this_statement,
+    sum(coalesce(ttl_disb_this_stmt ,0)) as total_disbursements_this_statement
+  from public.fec_vsum_f9_vw f
```

Comment:
@PaulClark2 wanted clarification on the most correct tables for electioneering.

We are using `electioneering_com_vw
` for line items and candidate totals https://github.com/18F/openFEC/blob/develop/data/sql_updates/electioneering.sql

We will want to use the same source of data for both.